docs/_template.html

```diff
@@ -2,84 +2,102 @@
-        <a href="/" class="d-flex align-items-center pb-3 mb-3 link-dark text-decoration-none border-bottom">
-            <svg class="bi me-2" width="30" height="24">
-                <use xlink:href="#bootstrap"/>
-            </svg>
-            <span class="fs-5 fw-semibold">Fantomas</span>
-        </a>
-        <a href="{{fsdocs-logo-link}}"><img id="fsdocs-logo" src="{{root}}/{{fsdocs-logo-src}}"/></a>
-        <!-- BEGIN SEARCH BOX: this adds support for the search box -->
-        <div id="header">
-            <div class="searchbox" id="fsdocs-searchbox">
-                <label for="search-by">
-                    <i class="fas fa-search"></i>
-                </label>
-                <input data-search-input="" id="search-by" type="search" placeholder="Search..."/>
-                <span data-search-clear="">
-                        <i class="fas fa-times"></i>
-                    </span>
+  <main id="main-container">
+    <div id="sidebar-wrapper" class="bg-primary h-100 collapse show" >
+      <div id="sidebar-logo" class="d-flex justify-content-center p-5">
+      <a href="{{fsdocs-logo-link}}"><img id="fsdocs-logo" src="{{root}}/{{fsdocs-logo-src}}" /></a>
+</div>
+      <ul class="list-unstyled ps-0 text-white" id="sidebar-generation">
+        {{fsdocs-list-of-documents}}
+        <!-- {{fsdocs-list-of-namespaces}} -->
+      </ul>
+    </div>
+    <div class="doc-content">
+      <div class="content-wrapper">
+     <nav class="navbar navbar-expand-lg border py-1">
+        <div class="container-fluid">
+          <button class="btn" data-bs-toggle="collapse" data-bs-target="#sidebar-wrapper" data-bs-parent="#main-container" aria-controls="sidebar-wrapper" aria-expanded="false" aria-label="Toggle navigation">
+            <i class="sidebar-toggle fs-3 bi bi-list"></i>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarText">
+            <div class="navbar-nav me-auto mb-2 mb-lg-0 fs-5">
+                <a class="nav-link active" aria-current="page" href="index.html">Docs</a>
+                <a class="nav-link" href="#">Tools</a>
             </div>
+            <form class="d-flex">
+              <div class="input-group me-4">
+              <label for="search-by">
+                <i class="fas fa-search"></i>
+              </label>
+                <div class="btn border-end-0 border">
+                  <i class="bi bi-search"></i>
+                </div>
+                <input aria-label="Search" class="form-control  border-start-0 border" data-search-input=""
+                  id="search-by" type="search" placeholder="Search..." />
+                <span class="input-group-append">
+                </span>
+                <span data-search-clear="">
+                  <i class="fas fa-times"></i>
+                </span>
+              </div>
+            </form>
+            <a class="nav-link m-0 p-0" href="https://github.com/fsprojects/fantomas">GitHub<i
```

Comment:
Use `target="_blank"` here, we want to keep the user on the documentation.
Same for the `tool` link.